plugins/grafana-custom-plugins/grafana-sankey-plugin/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "antreaflowvisibility-grafana-sankey-plugin",
```

Comment:
would there be any compatibility impact if we rename this to 'theia-grafana-sankey-plugin'?

Reply:
changed to 'theia-grafana-sankey-plugin'